src/components/Card.js

```diff
@@ -12,42 +12,37 @@ export default function Creepypasta({
   isBookmarked,
 }) {
   return (
-    <Grid>
-      <Wrapper>
-        <BookmarkButton
-          onBookmarkClick={onBookmarkClick}
-          isBookmarked={isBookmarked}
-        />
-        <Card
-          style={{
-            backgroundImage: `url(${image})`,
-            backgroundSize: 'cover',
-            backgroundRepeat: 'no-repeat',
-          }}
-        >
-          <CardContent>
-            <Title>{title}</Title>
-            <Wordcount>{wordcount} Words</Wordcount>
-          </CardContent>
-          <ReadMorePosition>
-            <RedButton>
-              <ReadMoreButtonText to="/fullcreepypasta" state={{ id: id }}>
-                Read
-              </ReadMoreButtonText>
-            </RedButton>
-          </ReadMorePosition>
-        </Card>
-      </Wrapper>
-    </Grid>
+    <Wrapper>
+      <BookmarkButton
+        onBookmarkClick={onBookmarkClick}
+        isBookmarked={isBookmarked}
+      />
+      <Card
+        style={{
+          backgroundImage: `url(${image})`,
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+        }}
+      >
+        <CardContent>
+          <Title>{title}</Title>
+          <Wordcount>{wordcount} Words</Wordcount>
+        </CardContent>
+        <ReadMorePosition>
+          <RedButton>
+            <ReadMoreButtonText to="/fullcreepypasta" state={{ id: id }}>
+              Read
+            </ReadMoreButtonText>
+          </RedButton>
+        </ReadMorePosition>
+      </Card>
+    </Wrapper>
   );
 }
 
-const Grid = styled.section`
+const Wrapper = styled.div`
```

Comment:
I think it would be more semantic to work with section instead of `div`. 🤔 But I think you could also work with `li` here, because you are rendering a list of creepypastas. If you change this, you should also adjust your CustomCard component to be more semantic.